src/database/migrations/20241004102426-create-connection-requests-table.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		await queryInterface.createTable('connection_requests', {
+			id: {
+				type: Sequelize.INTEGER,
+				autoIncrement: true,
+				primaryKey: true,
+			},
+			user_id: {
+				type: Sequelize.STRING,
+				allowNull: false,
+				primaryKey: true,
+			},
+			friend_id: {
+				type: Sequelize.STRING,
+				allowNull: false,
+				primaryKey: true,
+			},
+			status: {
+				type: Sequelize.STRING,
+				allowNull: false,
+			},
+			meta: {
+				type: Sequelize.JSON,
+			},
+
+			updated_by: {
+				type: Sequelize.STRING,
+				allowNull: false,
+			},
+			created_by: {
+				type: Sequelize.STRING,
+				allowNull: false,
+			},
+			created_at: {
+				type: Sequelize.DATE,
+				allowNull: false,
+			},
+			updated_at: {
+				type: Sequelize.DATE,
+				allowNull: false,
+			},
+			deleted_at: {
+				type: Sequelize.DATE,
+				allowNull: true,
+			},
+		})
+		await queryInterface.addIndex('connection_requests', ['user_id', 'friend_id'], {
```

Comment:
@nevil-mathew Similar comments as in the connections table migration.

Reply:
@nevil-mathew to check what should be the API response when User A sends a request to User B, and at the same time User B sends a request to user A